Makefile

```diff
@@ -359,6 +360,15 @@ ifeq ($(CAPSTONE_SHARED),yes)
 endif
 endif
 
+$(SUPPORT):
+	echo '#ifndef CAPSTONE_SUPPORT_H' > $@
+	echo '#define CAPSTONE_SUPPORT_H' >> $@
+	for a in $(CAPSTONE_ARCHS) ; do \
+		ARCH=$$(echo $$a|tr a-z A-Z); \
+		echo '#define CS_SUPPORT_ARCH_'$${ARCH} 1 >> $@ ; \
+	done
+	echo '#endif' >> $@
```

Comment:
i dont get the point of using cmake here. make is quite legacy and for a simple project like this we dont need cmake or autotools